src/new.spec.js

```diff
@@ -90,4 +92,26 @@ describe("withStore", () => {
       expect(ownProps(props)).toEqual(["bar"]);
     });
   });
+
+  describe("computed", () => {
+    it("can read state and computed and cannot write state and computed", () => {
+      const { getState } = makeTestInstance({
+        initialState: () => ({
+          foo: "bar",
+        }),
+        computed: {
+          qux: ({ foo }) => foo,
+          corge: ({ qux }) => qux,
+          thud: state => {
+            assert(isReadOnly(state));
+          },
+        },
+      });
+
+      noop(getState().thud);
```

Comment:
Why `noop`?

Reply:
To call the computed. Otherwise it won't be called and the test will be not triggered